docs/intro/install.md

```diff
@@ -40,6 +39,24 @@ Then run this command to move the `starport` executable to `/usr/local/bin/`:
 sudo mv starport /usr/local/bin/
 ```
 
+### Error while running command `protoc`
+
+If you get errors related to `protoc`, please, make sure that you followed [the instructions](https://grpc.io/docs/protoc-installation/#install-pre-compiled-binaries-any-os):
+
+1. Manually download from github.com/google/protobuf/releases the zip file corresponding to your operating system and computer architecture (`protoc-<version>-<os><arch>.zip`).
```

Comment:
```suggestion
1. From [github.com/google/protobuf/releases](github.com/google/protobuf/releases), manually download the zip file that corresponds to your operating system and computer architecture (`protoc-<version>-<os><arch>.zip`).
```

Reply:
Updated the commend to add `https://` to the link.